manifests/charts/base/files/gen-istio-cluster.yaml

```diff
@@ -3620,6 +3620,11 @@ rules:
   - apiGroups: [""]
     resources: ["secrets"]
     verbs: ["get", "watch", "list"]
+
+  #Used for MCS serviceexport management
+  - apiGroups: [""]
```

Comment:
Is this in core group, i think it can not be.

Reply:
I'm a bit unfamiliar with the manifests, could you recommend a more appropriate place?